keyboards/giabalanai/config.h

```diff
@@ -0,0 +1,247 @@
+ */
+#define SOFT_SERIAL_PIN D2
+#define SELECT_SOFT_SERIAL_SPEED 1
+/*Sets the protocol speed when using serial communication*/
+//Speeds:
+//0: about 189kbps (Experimental only)
+//1: about 137kbps (default)
+//2: about 75kbps
+//3: about 39kbps
+//4: about 26kbps
+//5: about 20kbps
+
+// Right side has to be the master since 1, LED data is output from right side, and 2, Audio pin is prepared on right side as a reserve.
+#define MASTER_RIGHT
+
+
+//#define BACKLIGHT_PIN B7
+//#define BACKLIGHT_LEVELS 3
+//#define BACKLIGHT_BREATHING
+
+/* ws2812 RGB LED */
+#define RGB_DI_PIN D3
+
+#ifdef RGBLIGHT_ENABLE
+
+#    define RGBLED_NUM 123
+// Do not define "RGBLED_SPLIT" since somehow it doesn't work well yet.
+// Even thhough "#define RGBLED_SPLIT { 60, 63 }" was set, LEDs on the sub keyboad side didn't turn on.
+// Not sure but rgblight_sethsv_at() might not support RGBLED_SPLIT yet.
+// Instead, LED data is tranferred from right side to the left via TRRS cable.
+
+#    define RGBLIGHT_LIMIT_VAL 100
+// #define RGBLIGHT_LAYERS
+
+#    ifndef HSV_BLACK
+#        define HSV_BLACK 0, 0, 0  // Define 'black' color, more like 'LED off' (H, S, V)
+#    endif
+
+// By default, LEDs of the buttons which are pressed turn on.
+// #    define RGBLIGHT_EFFECT_TWINKLE
+// #    define RGBLIGHT_EFFECT_KNIGHT
+
+
+// RGBLIGHT LED MAP
+//     Right 1st row (12 LEDs)
+//     ENCODER
+//     Right 2nd .. 5th row(13, 12, 13, 12 LEDs)
+//     Left 1st .. 5th row (12 LEDs x 5 rows)
+
+#ifdef MASTER_LEFT
+#    define RGBLIGHT_LED_MAP { \
+   11,  10,  9,   8,   7,   6,   5,   4,   3,   2,   1,   0,  \
+   12,  13,  14,  15,  16,  17,  18,  19,  20,  21,  22,  23, \
+   35,  34,  33,  32,  31,  30,  29,  28,  27,  26,  25,  24, \
+   36,  37,  38,  39,  40,  41,  42,  43,  44,  45,  46,  47, \
+   59,  58,  57,  56,  55,  54,  53,  52,  51,  50,  49,  48, \
+   \
+   60,  61,  62,  63,  64,  65,  66,  67,  68,  69,  70,  71, \
+   97,  \
+   84,  83,  82,  81,  80,  79,  78,  77,  76,  75,  74,  73,  72, \
+   85,  86,  87,  88,  89,  90,  91,  92,  93,  94,  95,  96,  \
+   110, 109, 108, 107, 106, 105, 104, 103, 102, 101, 100, 99,  98, \
+   111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122 \
+   }
+#    endif
+#ifdef MASTER_RIGHT
+#    define RGBLIGHT_LED_MAP { \
+   60,  61,  62,  63,  64,  65,  66,  67,  68,  69,  70,  71, \
+   97, \
+   84,  83,  82,  81,  80,  79,  78,  77,  76,  75,  74,  73,  72, \
+   85,  86,  87,  88,  89,  90,  91,  92,  93,  94,  95,  96, \
+   110, 109, 108, 107, 106, 105, 104, 103, 102, 101, 100, 99,  98, \
+   111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, \
+   \
+   11,  10,  9,   8,   7,   6,   5,   4,   3,   2,   1,   0,  \
+   12,  13,  14,  15,  16,  17,  18,  19,  20,  21,  22,  23, \
+   35,  34,  33,  32,  31,  30,  29,  28,  27,  26,  25,  24, \
+   36,  37,  38,  39,  40,  41,  42,  43,  44,  45,  46,  47, \
+   59,  58,  57,  56,  55,  54,  53,  52,  51,  50,  49,  48  \
+   }
+#    endif
```

Comment:
what about `EE_HANDS`? 

Also, I feel that rgb matrix may be better here, as it may be more appropriate for the board.

Reply:
I apologize, this part was for experimental purposes.
I'll delete it and make it more simple.
The method using EE_HANDS makes things more complicated in my opinion.
And probably it will be used as MASTER_RIGHT 100% in this keyboard so it is a waste to tweak here.

Regarding rgb matrix, I'm not sure how to realize turning on two LEDs at the same time when either one of the keys is pressed, as I explained here. https://github.com/qmk/qmk_firmware/pull/10125#discussion_r475501101

Could you help me if you have an easier way to do that than the current codes?
Thanks!

Reply:
@drashna, could you "Resolve conversation" if the above is OK?
This one, too. It's up and running exactly how I want to.
Thank you in advance!